artview/components/tools.py

```diff
@@ -58,6 +60,8 @@ def reset_tools(tooldict):
     -----
     Disconnects all tools.
     '''
+    import warnings
+    warnings.warn("this function is deprecated")
     for tool in tooldict:
         if tooldict[tool] is not None:
             tooldict[tool].disconnect()
```

Comment:
this is just a warning that this function is no longer being used.

I made however the opposite question, why remove from display class control over an internal part of it. There is no abstract concept of tool, it is an empiric construction, therefore any code separation is only visual.